modules/updating-control-plane-only-olm-operators.adoc

```diff
@@ -1,12 +1,12 @@
 // Module included in the following assemblies:
 //
-// * updating/updating_a_cluster/eus-eus-update.adoc
+// * updating/updating_a_cluster/control-plane-only-update.adoc
```

Comment:
Do we expect to be able to put redirects in place so that the older URLs work? Is there a docs policy on redirects?

Reply:
I've combed through the docs to find all mentions, including references in other sections :)